app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm.rb

```diff
@@ -33,6 +33,13 @@ class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::Vm < Man
   end
   supports :launch_html5_console
 
+  supports :native_console do
+    reason ||= _("VM Console not supported because VM is orphaned") if orphaned?
+    reason ||= _("VM Console not supported because VM is archived") if archived?
+    unsupported_reason_add(:native_console, reason) if reason
+  end
```

Comment:
I don't believe this is correct, when you're opening the "management console" of a VM it can be powered off

Reply:
Powered off VM can still open Management Console, it's `supported` by the above, no?

Reply:
What I try to do here is to hide the menu if we lost track of the VM's EMS or otherwise considered "archived" (=offline?) in ManageIQ?